README.md

```diff
@@ -68,10 +69,13 @@ $ pip install git+https://github.com/cogeotiff/rio-tiler-pds.git
 [s2_l2a_cog]: https://registry.opendata.aws/sentinel-2-l2a-cogs/
 [s1_l1c_cog]: https://registry.opendata.aws/sentinel-1/
 [l8_l1_cog]: https://registry.opendata.aws/landsat-8/
+[landsat_c2_cog]: https://www.usgs.gov/core-science-systems/nli/landsat/landsat-commercial-cloud-data-access
 [cbers_cog]: https://registry.opendata.aws/cbers/
 [modis_pds]: https://docs.opendata.aws/modis-pds/readme.html
 [modis_astraea]: https://registry.opendata.aws/modis-astraea/
 
+`*` Landsat 8 reader has been deprecated because the `landsat-pds` will be deleted on July 1st 2021.
```

Comment:
```suggestion
`*` Landsat 8 Collection 1 reader has been deprecated because the `landsat-pds` will be deleted on July 1st 2021. For new applications, using Collection 2 is suggested.
```